docs/index.md

```diff
@@ -1,34 +1,39 @@
 # Horizon Management Hub
 
-## <a name="deploy-all-in-one">Deploy All-in-One Horizon Management Hub, Agent and CLI</a>
+## <a id="deploy-all-in-one">Deploy All-in-One Horizon Management Hub, Agent and CLI</a>
```

Comment:
Can you revert this anchor tag to `<a name=`  ?  I fought this battle with `<a id=` several months ago. See git blame.  I think `name` is required for some post-processing.
https://github.com/open-horizon/devops/commit/fbfb128347db0b9cb33c9d38db18857673b7d1c7